website/docs/r/cluster.html.markdown

```diff
@@ -316,6 +323,12 @@ In addition to all arguments above, the following attributes are exported:
     - DELETED
     - REPAIRING
 
+### Plugin
```

Comment:
Since this is the resource I don't think we need this as it's sent but it's not set by the user.  It's set by the provider.    The databases doc should cover explaining it.